layouts/shortcodes/release-branches.html

```diff
@@ -0,0 +1,45 @@
+<h2 id="detailed-release-history-for-active-branches">Detailed Release History for Active Branches</h2>
+{{ range $releaseInfo := .Site.Data.releases.schedule.schedules }}
+{{- $version := printf "%.2f" $releaseInfo.release -}}
+{{- $dataIdVersion := replace ( printf "%.2f" $releaseInfo.release ) "." "-" | anchorize -}}
+
+<h3 id="{{ $dataIdVersion }}">{{ $version }}</h3>
+<p>
+    Next patch release is <strong>{{ $releaseInfo.next.release }}</strong>. {{ $releaseInfo.next.note }}
+</p>
+<p>
+    <strong>{{ $version }}</strong> enters maintenance mode on <strong><time datetime="{{ time.Format "2006-01-02" $releaseInfo.endOfLifeDate }}">{{ time.Format ( T "release_date_format") $releaseInfo.maintenanceModeStartDate }}</time></strong> and End of Life is on <strong><time datetime="{{ time.Format "2006-01-02" $releaseInfo.endOfLifeDate }}">{{ time.Format ( T "release_date_format") $releaseInfo.endOfLifeDate }}</time></strong>
+</p>
+{{ if $releaseInfo.previousPatches }}
+<div>
+    <table class="table table-striped col-md-8">
+        <thead class="thead">
+            <tr>
+                <th class="col-md-2">PATCH RELEASE</th>
+                <th class="col-md-2">CHERRY PICK DEADLINE</th>
+                <th class="col-md-2">TARGET DATE</th>
+                <th class="col-md-4">NOTE</th>
+            </tr>
+        </thead>
```

Comment:
We can make this change, but we're then switching from something which is _easy_ to localize to something that is _hard_ to localize.

Also, for all-uppercase text, write it in title case and then use CSS `text-transform` to make it caps if you wish.

Reply:
I am okay with keeping the table header title case. I wrote the shortcode in line with the existing page.

Reply:
> I am okay with keeping the table header title case.

It can be title case as rendered (I'd prefer sentence case anyway) - but for a shortcode, use CSS to achieve the transformation to upper case. It CSS doesn't feel right, use the `upper` function and keep the localizable string in sentence case.